### PR TITLE
Typo/0.10.5 fixes

### PR DESCRIPTION
**Pull request by TheGiraffe3**
Fixes a typo or two in map.txt, fixes some things that were in the wrong order, and adds the systems/planets from 0.10.5 to the map reveal.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
@warp-core can this be merged sooner rather than later?
~I don't want merge conflicts with #71~

File: data/map reveal.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
 outfit "Galaxy Map"
-	thumbnail "outfit/map"
-	map 1000
-	cost 0
 	category "Special"
+	cost 0
+	map 10000
+	thumbnail "outfit/map"
```
Comment (warp-core) [inline data/map reveal.txt]: It you're trying to match the order of the local map in the vanilla game, the thumbnail should come before the "map" attribute.

File: data/map reveal.txt [hunk 2]
```diff
@@ -12,19 +12,19 @@ planet "New Boston"
 
 
 mission "Map Reveal"
-	description `Reveal the entire map in one go! Accept this job and launch and all vanilla systems and planets will be visited.`
 	job
 	repeat
+	description "Reveal the entire map in one go! Accept this job and all vanilla systems and planets will be visited."
 	source
```
Comment (warp-core) [inline data/map reveal.txt:18]: The other missions in this plugin put the description after the source.

File: data/map reveal.txt [hunk 3]
```diff
@@ -12,19 +12,19 @@ planet "New Boston"
...
-		event "VISIT_EVERYTHING"
+		event "VISIT_EVERYTHING" 0
 		fail
 
 
 
-# Commented out line indicate defined planets that are not assigned to any objects at the start of the game.
-# They cannot be visited and attemmpting to do so renders this event invalid.
+# Commented out lines indicate defined planets that are not assigned to any objects at the start of the game.
+# They cannot be visited and attempting to say they are renders this event invalid.
```
Comment (warp-core) [inline data/map reveal.txt]:
```suggestion
# They cannot be visited and attempting to do so renders this event invalid.
```
I don't see anything wrong with this part.

**Comment by TheGiraffe3** [comment]
@warp-core any updates?

**Comment by TheGiraffe3** [comment]
@warp-core could you tell me if anything else needs to be changed about this?

File: data/map reveal.txt [hunk 4]
```diff
@@ -12,19 +12,19 @@ planet "New Boston"
 
 
 mission "Map Reveal"
-	description `Reveal the entire map in one go! Accept this job and launch and all vanilla systems and planets will be visited.`
 	job
 	repeat
 	source
 		attributes forge "map reveal"
+	description "Reveal the entire map in one go! Accept this job and all vanilla systems and planets will be visited."
```
Comment (warp-core) [inline data/map reveal.txt]: This ought to be between `repeat` and `source`.

**Pull request merged**